tags.json

```diff
@@ -2286,6 +2286,10 @@
     "weather",
     "light"
   ],
+  "sun-dim": [
+    "dim sun",
```

Comment:
```suggestion
    "brightness",
```

(remove "dim sun" tag as it's not really useful, and add "brightness")

Reply:
Thanks 🙂